drivers/display/ssd1673.c

```diff
@@ -363,11 +320,8 @@ static int ssd1673_set_brightness(const struct device *dev,
 
 static int ssd1673_set_contrast(const struct device *dev, u8_t contrast)
```

Comment:
It was a nice hack for initial update, why are you removing it? and why in this commit?

Reply:
It was no longer needed as the initial clear is done at init

Reply:
This commit must be reverted because it removed possibility for the user to perform initial update.